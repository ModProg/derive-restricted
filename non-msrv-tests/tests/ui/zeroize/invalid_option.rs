extern crate zeroize_ as zeroize;

#[derive_where::derive_where(Zeroize(test); T)]
struct Test1<T>(T);

#[derive_where::derive_where(Zeroize(test = "test"); T)]
struct Test2<T>(T);

fn main() {}
