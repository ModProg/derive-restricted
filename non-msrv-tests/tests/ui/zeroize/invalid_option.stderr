error: `Zeroize` doesn't support this option
 --> tests/ui/zeroize/invalid_option.rs:3:38
  |
3 | #[derive_where::derive_where(Zeroize(test); T)]
  |                                      ^^^^

error: `Zeroize` doesn't support this option
 --> tests/ui/zeroize/invalid_option.rs:6:38
  |
6 | #[derive_where::derive_where(Zeroize(test = "test"); T)]
  |                                      ^^^^^^^^^^^^^
